src/plugins/@autoattack.c

```diff
@@ -31,7 +31,7 @@
 #include "plugins/HPMHooking.h"
 #include "common/HPMDataCheck.h"
 
-#define OPTION_AUTOATTACK 0x10000000
+#define OPTION_AUTOATTACK 0x30000000
```

Comment:
Misclick the merge button, but shouldn't this be 0x2 or 0x4?

Reply:
yeah, my bad, fixed it in a new PR, sorry 😅